pkg/distributor/push_test.go

```diff
@@ -1244,6 +1244,25 @@ func TestOTLPPushHandlerErrorsAreReportedCorrectlyViaHttpgrpc(t *testing.T) {
 			expectedGrpcErrorMessage: "rpc error: code = Code(400) desc = ReadObjectCB: expect { or n, but found i, error found in #1 byte of ...|invalid|..., bigger context ...|invalid|...",
 		},
 
+		"invalid JSON with non-utf8 characters request returns 400": {
+			request: &httpgrpc.HTTPRequest{
+				Method: "POST",
+				Headers: []*httpgrpc.Header{
+					{Key: "Content-Type", Values: []string{"application/json"}},
+				},
+				Url:  "/otlp",
+				Body: []byte("\n\xf6\x16\n\xd3\x02\n\x1d\n\x11container.runtime\x12\x08\n\x06docker\n'\n\x12container.h"),
```

Comment:
I'm pretty sure that the test is correct, but it would be easier to read and understand it if it had a comment explaining which character is not valid UTF-8 and with which other character we expect it to be replaced.

Reply:
I have added a comment explaining which characters are non-UTF8, and what should they be replaced with. Thank you.